sdk/test-utils/recorder-new/test/testProxyTests.spec.ts

```diff
@@ -259,7 +259,7 @@ function getTestServerUrl() {
         );
       });
 
-      it.skip("ContinuationSanitizer", async () => {
+      it("ContinuationSanitizer", async () => {
```

Comment:
The intention behind this sanitizer is to handle a case at ACS service.
Right now, the proxy tool only supports continuation sanitizer only in the headers I think.
I (vaguely) remember the ACS case to have the id at request body.

If the sanitizer can't be used for that case, it's pretty much worthless.
Can you test that case on your local, observe the changes in the recordings to understand, and then see what else we'd need to add in the proxy tool to support such a case?
@jeremymeng should be able to point you to the test case from ACS.

After you test using the ACS test, add a new test or update this test to represent the exact scenario over there.
Add more/different routes to the server under utils as needed.

Reply:
the name `Continuation` doesn't ring a bell. In ACR I had to sanitize part of the token value while keeping expiration info.  I also had to call `pluginForClientSecretCredentialTests(env.CONTAINERREGISTRY_TENANT_ID)` but I don't remember the exact reason now.

https://github.com/Azure/azure-sdk-for-js/blob/c07af4484e08f4407d284ed6833c7978524835bb/sdk/containerregistry/container-registry/test/utils/utils.ts#L41-L57

Reply:
@jeremymeng this is about some UUID that is present as part of the response of one request, is passed as part of a future request of the same test.

Reply:
Talked to Jeremy, and turns out this wasn't the requirement.
Let's move on with what we have and revisit if we need to support something else while migrating the tests.